deploy/certified-metadata-bundle/cockroach-operator/1.2.11/cockroach-operator.v1.2.11.clusterserviceversion.yaml

```diff
@@ -34,14 +34,14 @@ metadata:
     capabilities: Full Lifecycle
     categories: Database
     certified: "true"
-    containerImage: us.gcr.io/chris-love-operator-playground/cockroachdb-operator:v1.8.37
-    createdAt: "2021-03-11T18:15:39Z"
+    containerImage: us.gcr.io/chris-love-operator-playground/cockroachdb-operator:v1.8.48
```

Comment:
We should not be checking in generated files that point to chris's dev workspace